microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -750,6 +742,16 @@ Microsoft Edge WebView controls allow app developers to host web content in nati
 
 To enable diagnostic data collection, go to `edge://settings/privacy`. Turn on the **Optional diagnostic data** setting. To turn off diagnostic data collection for Microsoft Edge on Windows 10 and later, open **Start** > **Settings** > **Privacy** and select **Diagnostics & feedback**. To turn off diagnostic data collection for all other platforms, go to `edge://settings/privacy`. Turn off the **Help improve Microsoft products by sending optional diagnostic data about how you use the browser, websites you visit, and crash reports** setting. The applications hosting Microsoft Edge WebView may collect other data that is governed by the data collection management of the developer and relevant privacy policies.
 
+<!-- ====================================================================== -->
+## Writing assistance
+
+Microsoft Edge provides enhanced spelling and grammar checking as you type in the browser. Go to `edge://settings/languages` and select **Microsoft Editor** under **Use writing assistance toggle**. If you select **Microsoft Editor** to provide enhanced spelling and grammar checking, Microsoft Edge will send your typed text and a service token to a Microsoft cloud service over a secure HTTPS connection. The service token doesn't contain any user-identifiable information. A Microsoft cloud service then processes the text to detect spelling and grammar errors in your text. All your typed text sent to Microsoft are deleted immediately after processing occurs. No data is stored for any period of time. If you select **Basic** under **Use writing assistance** toggle in `edge://settings/languages`, the service will perform only local spellchecking on the device and no data will be sent to the cloud for spellchecking services. 
+
+To turn off all spelling and grammar capabilities, toggle **Use writing assistance toggle** off. 
+ 
+When you add a new language to Microsoft Edge, the browser downloads the dictionary for the new language to the device using HTTPS. The dictionary is used for the basic spellcheck service or for languages not supported by Editor spelling and grammar checking. Deleting the language from your Microsoft Edge settings deletes the dictionary from the device.To add or remove words in your local dictionary, go to `edge://settings/languages` and under **Customize dictionary**, select **Add or delete words**.
+
+ Custom dictionary is not available in **Guest** mode.
 
```

Comment:
fyi feel free to add h4 headings, skipping h3's, to give more skimmable points of entry and break up long sections - like labelling paragraphs to differentiate their main focus.